pkgs/development/python-modules/afdko/default.nix

```
@@ -1,22 +1,32 @@
-{ stdenv, buildPythonPackage, fetchPypi, pythonOlder, python
+{ stdenv, buildPythonPackage, fetchPypi, fetchpatch, pythonOlder, python
 , fonttools, defcon, lxml, fs, unicodedata2, zopfli, brotlipy, fontpens
 , brotli, fontmath, mutatormath, booleanoperations
-, ufoprocessor, ufonormalizer, psautohint
+, ufoprocessor, ufonormalizer, psautohint, tqdm
 , setuptools_scm
 , pytest
 }:
 
 buildPythonPackage rec {
   pname = "afdko";
-  version = "3.5.0";
+  version = "3.5.1";
 
   disabled = pythonOlder "3.6";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "0wid4l70bxm297xgayyrgw5glhp6n92gh4sz1nd4rncgf1ziz8ck";
+    sha256 = "1qg7dgl81yq0sp50pkhgvmf8az1svx20zmpkfa68ka9d0ssh1wjw";
   };
 
+  # Skip date-dependent test. See
+  # https://github.com/adobe-type-tools/afdko/pull/1232
+  # https://github.com/NixOS/nixpkgs/pull/98158#issuecomment-704321117
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/adobe-type-tools/afdko/pull/1232.patch";
```

please use the commit patch
```suggestion
      url = "https://github.com/adobe-type-tools/afdko/commit/2c36ad10f9d964759f643e8ed7b0972a27aa26bd.patch";
```

---

done